charts/incubator/spotweb/values.yaml

```
@@ -0,0 +1,56 @@
+image:
+  repository: jbaten/spotweb
+  pullPolicy: IfNotPresent
+  tag: 1.5.1@sha256:1008eeb207c326f5e1af86058d9591771fa9b111c188544467d42c4862b60fd7
+
+# See more environment variables in the babybuddy documentation
+# https://hub.docker.com/r/jgeusebroek/spotweb
+env:
+  TZ: UTC
+  SPOTWEB_SYSTEMTYPE: single
+  SPOTWEB_FIRSTNAME: demo
+  SPOTWEB_LASTNAME: spotweb
+  SPOTWEB_MAIL: demo@spotweb.com
+  SPOTWEB_RETRIEVE: "15min"
+  DB_ENGINE: "pdo_pgsql"
+  DB_PORT: "5432"
+  DB_DATABASE: "{{ .Values.postgresql.postgresqlDatabase }}"
+  DB_USER: "{{ .Values.postgresql.postgresqlUsername }}"
+  DB_SCHEMA: "public"
+
+envValueFrom:
+  DB_PASSWORD:
+    secretKeyRef:
+      name: dbcreds
+      key: postgresql-password
+  DB_HOST:
+    secretKeyRef:
+      name: dbcreds
+      key: plainhost
+
+secret:
+  SPOTWEB_USERNAME: myawesomeuser
+  SPOTWEB_PASSWORD: changeme
+
+securityContext:
+  readOnlyRootFilesystem: false
+  allowPrivilegeEscalation: false
+  runAsNonRoot: false
```

```suggestion
securityContext:
  readOnlyRootFilesystem: false
  runAsNonRoot: false
```
Have you tried without it? Not sure it needs that

---

Yeah, I tried readonly filesystem and running as non root. But the container needs it at the moment. PHP-FPM otherwise throws error trying to open log files.

---

I was talking only for priv esca mostly :D